### ISSUE TITLE
make build - broken on a fresh image build

### ISSUE THREAD

**Issue by amcmanus** [votingworks/vxsuite-complete-system#268]
The addition of build support for the new `apps` structure in vxsuite has introduced a subtle bug only seen when building on a new system.

tl;dr - the easiest fix is to add the packages included in `services/smartcards` `make install` as part of the initial `make deps` step when building vxsuite-complete-system on an image for the first time.

More details:

The latest version of build.sh starts with `apps/vx-scan`. 
Once it hits this command: `BUILD_ROOT="${BUILD_ROOT}/vxsuite" ./script/prod-build`, it invokes a pip dependency that isn't met b/c the `services/smartcards` system packages don't exist since smartcards hasn't been built yet. (Fully admit I don't understand the pip dependency bit in depth, but it's definitely there.)

Note: You'll never encounter this bug on a system that had already installed the packages in `services/smartcards`.

Longer term, I think we should reconsider how we manage system-level packages, along with the `build.sh` script. I think our system's build complexity warrants breaking this script into smaller, more manageable/explicit pieces rather than attempting to meet all needs in a single script. It is a non-trivial exercise to work through the script, especially as our list of systems grow, and the lack of explicit ordering can lead to subtle bugs like this one.
